sample/kotlin/src/main/kotlin/com/datadog/android/sample/data/Result.kt

```diff
@@ -0,0 +1,12 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sample.data
+
+sealed class Result {
```

Comment:
Since you rewrote this class, maybe we can reuse it in the `TracesViewModel` class too ?

Reply:
Oh...I didn't see you had one there, I will use this there then also.